instructor/ui/instructor.js

```diff
@@ -489,10 +474,80 @@ $(function() {
       paper.view.update();
 
     });
+
     load_sketch_tool(x_upper_bound, tick_values);
+
+
+    mypapers[3].setup("noise-canvas");
+
+    /*
+      Preview noise for sketched graph
+     */
+    $('.scb_ab_f_add_noise').click(function () {
+      /* Can preview noise, when graph has been plotted */
+      if (typeof(data_points) != 'undefined' && data_points.length > 0) {
+        if ($(".scb_ab_s_preview_canvas_div").css('display') == 'none') {
+          $(".scb_ab_s_preview_canvas_div").show();
+          $(".scb_ab_f_add_noise").text('CLOSE');
+          mypapers[3].activate();
+          paper.project.clear();
+          paper.project.view.viewSize.width = 450;
```

Comment:
Can you create variables for `450` and `250` and reuse them below when defining `fraction` and `y_fraction`?